src/search/airportsearch.cpp

```diff
@@ -799,7 +799,7 @@ void AirportSearch::randomFlightClicked(bool showDialog)
   {
     // Disable button to avoid multiple clicks
     ui->pushButtonAirportFlightplanSearch->setDisabled(true);
-    // TODO: lock predefined airport ui
+    // TODO: lock predefined airport ui.
```

Comment:
the predefined indices are accessed on result thus thay shouldn't change between start and end. but you abstracted them away I think so you can ignore this.